core/src/test/kotlin/fr/sncf/osrd/pathfinding/PathfindingV2Test.kt

```diff
@@ -134,7 +134,20 @@ class PathfindingV2Test : ApiTest() {
             parsed.incompatibleConstraints ==
                 IncompatibleConstraints(
                     incompatibleElectrificationRanges =
-                        listOf(RangeValue(Pathfinding.Range(Offset.zero(), Offset.zero()), "elec")),
+                        listOf(
+                            RangeValue(
```

Comment:
Throwback to previous comment, feels a bit scuffed to not just have one unique `RangeValue` going from 125 to 10250.

Reply:
Indeed, it was just a first try, better in amended https://github.com/OpenRailAssociation/osrd/pull/7953/commits/8ee6553e87735672c30538f445ee6e66d877